src/Commands/SyncCommand.php

```diff
@@ -115,4 +170,20 @@ private function buildDefault(string $type, $default)
 
         return $default;
     }
+
+    private function validateUpdateFields(): void
+    {
+        if (! $this->option('update-fields')) {
+            return;
+        }
+
+        $fields = explode(',', $this->option('update-fields'));
+        
+        foreach ($fields as $field) {
+            throw_if(
+                (! in_array($field, $this->availableFieldsToUpdate)),
```

Comment:
_No hace falta el paréntesis de afuera._

Acá se puede evitar la negación: `throw_unless(in_array($field, $this->availableFieldsToUpdate), ...)`